xapian-core/tests/testdata/apitest_cluster.txt

```diff
@@ -0,0 +1,7 @@
+A computer cluster consists of a set of loosely or tightly connected computers that work together so that, in many respects, they can be viewed as a single system. Unlike grid computers, computer clusters have each node set to perform the same task, controlled and scheduled by software.
```

Comment:
This is a bit weird — in a clustering system, having text about a totally different sort of cluster! It's probably not a problem, but isn't really an ideal choice of text.